### PR TITLE
Plugin inlining (backport)

### PR DESCRIPTION
**Pull request by rhuss**
This is a backport of #902

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -41,6 +41,10 @@
 | 🐛
 | fix(tekton e2e): Refer tasks from new tekton catalog task structure
 | https://github.com/knative/client/pull/966[#966]
+
+| 🎁
+| Add support for internal plugins
+| https://github.com/knative/client/pull/880[#880]
```
Comment (navidshaikh) [inline CHANGELOG.adoc:47]: lets update the date of the release (v0.16.1) as well ?

**Review by navidshaikh — APPROVED** [review]

File: CHANGELOG.adoc (hunk 1 above)
Comment (navidshaikh) [reply CHANGELOG.adoc:47]: @rhuss : also the PR number is #902

Comment (rhuss) [reply CHANGELOG.adoc:47]: ah right. going to fix it now.

**Review by navidshaikh — APPROVED** [review]
/lgtm

**Pull request merged**